grapher/core/Grapher.tsx

```diff
@@ -1864,9 +1861,16 @@ export class Grapher
     }
 
     private disableAutoFaceting = true // turned off for now
+
+    // the actual facet setting used by a chart, potentially overriding selectedFacetStrategy
     @computed get facetStrategy(): FacetStrategy {
-        if (this.facet && this.availableFacetStrategies.includes(this.facet))
-            return this.facet
+        if (!this.showFacets) return FacetStrategy.none
```

Comment:
An author might want to set a faceting strategy but not allow readers to edit it?

We do this for example for the timeline – an author can change the timeline, then hide it (`hideTimeline`), and it stays applied for readers.

Reply:
Agreed, we want to support that. I renamed the flag to `hideFacetControl` and made the faceting stick even if the control is hidden.